src/utils.ts

```diff
@@ -30,6 +30,30 @@ class Utils {
     );
   }
 
+  /**
+   * Validate if the given string is a valid X-address on the XRP Ledger.
```

Comment:
nit: prefer `Validate whether the given string is a valid X-address for the XRP Ledger.`

`on the XRP Ledger` implies that the function is querying the ledger itself to see whether the account exists there, but this is just an offline check